packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,75 @@ return [
 ];
 ```
 
+## Usage
```

Comment:
```suggestion
## Usage

By default this bundle assumes you are using [Gedmo Sortable](https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/doc/sortable.md) to handle the sort order of your entities, so unless you are using another method or want to change the default field name `position`, you don't need to configure anything for the bundle.

We provide a trait so you can easily add the position field with the Gedmo configuration on each entity you want to be able to sort:

```php

namespace App\Entity;

use Runroom\SortableBehaviorBundle\Behaviors\Sortable;

class MyEntity
{
    use Sortable;
    // ... rest of your class
}
```